src/server/mod.rs

```diff
@@ -141,7 +141,7 @@ macro_rules! build_app {
             // XXX: This route will be enabled when we are ready to roll out Tokenserver
             // Tokenserver
             // .service(
-            //     web::resource("/1.0/sync/1.5".to_string())
```

Comment:
I added support for both `sync-1.1` and `sync-1.5` for now. We may remove the `services` table entirely, at which point we would be able to remove this